server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -44,25 +44,6 @@
  */
```

Comment:
I think we should also update the java doc, replace old example like` IBP_0_10_0-IV0` to  `IBP_3_3_IV3`

Reply:
Good point - I can also simplify the parsing code since we no longer have metadata versions starting with `0.`.